client/js/InitVue.js

```diff
@@ -40,6 +41,7 @@ Vue.use(DPVueCorePlugin)
 
 // Register components that are used globally
 Vue.component('DpObscure', DpObscure)
+Vue.component('RegisterFlyout', RegisterFlyout)
```

Comment:
We could also register it in line 70.

Reply:
Done in [b700e9c](https://github.com/demos-europe/demosplan-core/pull/2366/commits/b700e9c45352bf30fcfb77b3dd5a4cd9b361e297)